### ISSUE TITLE
Milestone 3 - Data Schema: Missing Submission!

### ISSUE THREAD

**Issue by minhkhang1795** [FitPeople/FitNote#3]
It appears that you haven't defined your data schema. Please have your project updated as soon as possible!

Best,
Khang

**Comment by dan-choe** [comment]
Sorry for late for creating an issue.
We actually added optional features and api what we will use in the app.

**Comment by minhkhang1795** [comment]
Sorry, maybe you did not understand my previous comment. I would like to see your Parse server set up and defined tables with appropriate columns for your models (see example below). Then you should add these to your README file and commit your project to the latest version.

**For instance:**
Data models:
- User: id, name, DOB, email,...
- Location: lat, long,...

Sorry for my late reply! You guys should continue working on your amazing idea and turn it into a real fancy app. :+1:

**Comment by dan-choe** [comment]
Hello,
Do we have to set up Parse server even though we don't need to use?

**Comment by minhkhang1795** [comment]
Yes, if your project does not need Parse server, you don't have to. However, you need to define your app models :smile:
